packages/shared/tsconfig.json

```diff
@@ -0,0 +1,26 @@
+{
+  "extends": "@tsconfig/svelte/tsconfig.json",
+  "compilerOptions": {
+    "target": "esnext",
+    "module": "esnext",
+    "strict": true,
+    "resolveJsonModule": true,
+    "preserveValueImports": false,
+    "baseUrl": ".",
+    /**
+     * Typecheck JS in `.svelte` and `.js` files by default.
+     * Disable checkJs if you'd like to use dynamic types in JS.
+     * Note that setting allowJs false does not prevent the use
+     * of JS in `.svelte` files.
+     */
+    "allowJs": true,
+    "checkJs": true,
+  },
+  "include": [
+    "src/**/*.d.ts",
+    "src/**/*.ts",
+    "src/**/*.js",
+    "types/*.d.ts",
+    "../../types/**/*.d.ts",
+  ]
```

Comment:
shared should not have any svelte imports (basically shared can't have dom or node types it's shared between server side and client side)

Reply:
I tried modifying this but gave up on the erroring out, I implemented based on your PR a while back: https://github.com/projectatomic/ai-studio/commit/0ea4d2135f4e15402d4bfc07ddc3e43c8fa136a1#diff-28fbbd10733902283f23978b34aacb8284bc7af50d729f6d0ea1526ba625d322

Any suggestions how we could modify this / still have it pass / make the changes too to aistudio?

Reply:
I still don't see why shared has svelte in its tsconfig file